source/install/troubleshooting.rst

```diff
@@ -19,9 +19,41 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Review Mattermost Logs
```

Comment:
These changes were merged separately, so we can remove these changes from this PR.

Reply:
should be solved